src/main/kotlin/thatline/localup/auth/service/EmailService.kt

```
@@ -14,26 +16,35 @@ class EmailService(
-            to = email,
-            subject = VERIFICATION_SUBJECT,
-            htmlContent = htmlContent
-        )
+            sendEmail(
+                to = email,
+                subject = VERIFICATION_SUBJECT,
+                htmlContent = htmlContent
+            )
+            
+            logger.info("Verification email sent successfully to: $email")
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)